src/main/java/org/opendatanode/plugins/extractor/ckan/file/FilesFromCkan.java

```diff
@@ -11,26 +11,30 @@
 import eu.unifiedviews.dataunit.files.WritableFilesDataUnit;
 import eu.unifiedviews.dpu.DPU;
 import eu.unifiedviews.dpu.DPUContext;
-import eu.unifiedviews.dpu.DPUException;
 import eu.unifiedviews.dpu.DPUContext.MessageType;
+import eu.unifiedviews.dpu.DPUException;
 import eu.unifiedviews.helpers.dataunit.files.FilesDataUnitUtils;
 import eu.unifiedviews.helpers.dpu.config.ConfigHistory;
 import eu.unifiedviews.helpers.dpu.context.ContextUtils;
 import eu.unifiedviews.helpers.dpu.exec.AbstractDpu;
+import eu.unifiedviews.plugins.extractor.ckan.file.FilesFromCkanConfig_V1;
 
 @DPU.AsExtractor
 public class FilesFromCkan extends AbstractDpu<FilesFromCkanConfig_V1> {
 
     private static final Logger LOG = LoggerFactory.getLogger(FilesFromCkan.class);
 
-    public static final String CONFIGURATION_SECRET_TOKEN = "dpu.uv-e-filesFromCKAN.secret.token";
-    public static final String CONFIGURATION_CATALOG_API_LOCATION = "dpu.uv-e-filesFromCKAN.catalog.api.url";
+    public static final String CONFIGURATION_SECRET_TOKEN = "org.opendatanode.CKAN.secret.token";
+
+    public static final String CONFIGURATION_CATALOG_API_LOCATION = "org.opendatanode.CKAN.api.url";
+
```

Comment:
Usage of this parameter implemented as is in L-*CKAN DPUs
see:
* https://github.com/OpenDataNode/UVPlugin-filesToCKAN/blob/develop/src/main/java/eu/unifiedviews/plugins/loader/filestockan/FilesToCkan.java#L105
* https://github.com/OpenDataNode/UVPlugin-filesToCKAN/blob/develop/src/main/java/eu/unifiedviews/plugins/loader/filestockan/FilesToCkan.java#L148
for more details